### ISSUE TITLE
MIRACLRetrieval results are missing for most models.

### ISSUE THREAD

**Issue by Samoed** [embeddings-benchmark/mteb#1550]
Models results for `MIRACLRetrieval` are missing for `MTEB(rus)`

**Comment by KennethEnevoldsen** [comment]
Do they need to be run or missing from the leaderboard?

**Comment by Samoed** [comment]
They are missing, but they were present in the old version of the leaderboard

**Comment by KennethEnevoldsen** [comment]
Hmm so we need to check the results repo and if they are not there probably rerun them

**Comment by Samoed** [comment]
It seems strange. In the new leaderboard, only `MIRACRLERetrieval` is listed, but the benchmark includes both `MIRACRLERetrieval` and `MIRACRLEReranking`. I think the `MIRACLReranking` scores are missing because they were previously labeled with a suffix like `(MIRACLE)` (e.g., `NDCG@10(MIRACL)`). However, I don't understand why `MIRACRLERetrieval` is also missing. 

Results for these datasets are available in the results repository, such as for `bge-m3`:  
- [MIRACRLERetrieval](https://github.com/embeddings-benchmark/results/blob/main/results/BAAI__bge-m3/5617a9f61b028005a4858fdac845db406aefb181/MIRACLRetrieval.json)
- [MIRACLReranking](https://github.com/embeddings-benchmark/results/blob/main/results/BAAI__bge-m3/5617a9f61b028005a4858fdac845db406aefb181/MIRACLReranking.json)

https://github.com/embeddings-benchmark/mteb/issues/1317#issuecomment-2504696294

![image](https://github.com/user-attachments/assets/e7f2cccd-f789-4171-8dd1-3e09969e4b18)

https://github.com/embeddings-benchmark/mteb/blob/d713525824ab94e6269dad6dab2911c9bf71296c/mteb/benchmarks/benchmarks.py#L245-L248

**Comment by KennethEnevoldsen** [comment]
Sounds like we need to either rerun MICRACL or rewrite the scores.

> However, I don't understand why MIRACRLERetrieval is also missing.

@Samoed isn't it because the main score has changed?

**Comment by Samoed** [comment]
No, `MIRACLRetrieval` has scores as all retrieval tasks. Only `MIRACRLReranking` has different score names

**Comment by x-tabdeveloping** [comment]
I have a suspicion, that this might be because of duplicate results, where the incorrect one gets used in the leaderboard. Can't confirm yet though, I'll have a look

**Comment by x-tabdeveloping** [comment]
Wait a sec, bge-m3 shouldn't even show up in the leaderboard, since it has no metadata, and it doesn't for me. Are you sure it's bge-m3 we're looking at? @Samoed

**Comment by Samoed** [comment]
I've added bge-m3 just to show that in result repository some scores for tasks are presented

**Comment by x-tabdeveloping** [comment]
If I load the results then I only get MIRACL scores for one model, even when I don't filter models out that don't have metadata.
```python
import mteb

all_results = mteb.load_results()
miracl = all_results.filter_tasks(["MIRACLRetrieval"])
print(len(miracl.model_results))
# 1
print(miracl[0])
# model_name='jinaai/jina-embeddings-v3' model_revision='215a6e121fa0183376388ac6b1ae230326bfeaed' task_results=[TaskResult(task_name=MIRACLRetrieval, scores=...)]
```

**Comment by x-tabdeveloping** [comment]
so something is either wrong with the results or with the result loading script

**Comment by x-tabdeveloping** [comment]
It seems that we only have results on the "dev" split on MIRACLRetrieval for intfloat/multilingual-e5-large-instruct. Maybe this could be the source of our issues?

**Comment by x-tabdeveloping** [comment]
Also, when I load results, I get these warnings for almost all models:
```
Validation failed for MIRACLRetrieval in sentence-transformers/all-MiniLM-L6-v2 8b3219a92973c328a8e22fadcfa821b5dc75636a: Missing subsets {'fi', 'sw', 'te', 'fa', 'ko', 'zh', 'de', 'yo', 'e
n', 'ar', 'es', 'th', 'fr', 'hi', 'id', 'bn', 'ja'} for split dev
```

**Comment by x-tabdeveloping** [comment]
Maybe the loading function should be a bit more graceful, when stuff is missing

**Comment by x-tabdeveloping** [comment]
Raising warnings instead of errors when loading results and languages are missing partly fixes the issue.

**Comment by KennethEnevoldsen** [comment]
How

**Comment by x-tabdeveloping** [comment]
right.. I mean we are still missing a lot of results on MIRACL, just not the Russian ones
